commons/dto/DonationDTO.ts

```diff
@@ -19,7 +19,7 @@ export type DonationDTO = DTO & HasIdentifier & {
   geohash: string;
   donationDateTime: string;
   status: DonationStatus;
-  contactInfo: { name: string; phone: string };
+  contactNumber: string;
   patientName?: string;
```

Comment:
if this is optional, it should be a separate data type that extends the core data type.

Reply:
the reason being, what if i want to force the type casting on a particular object? then I can't guarantee the validity of the object

Reply:
I see your point about separating types for clarity.

If I understand you correctly, you’re suggesting that if we keep required and optional fields in one type, we can't guarantee that the required fields will be present when we forcefully cast an object.

However, if we separate the required fields from the optional ones and then extend the required type, I believe we still cannot guarantee the validity of the object.